src/avram/form_errors.cr

```diff
@@ -0,0 +1,12 @@
+module Avram::FormErrors
+  def errors
+    fields.reduce({} of Symbol => Array(String)) do |errors_hash, field|
+      if field.errors.empty?
+        errors_hash
+      else
+        errors_hash[field.name] = field.errors
+        errors_hash
+      end
+    end
+  end
```

Comment:
Can you add an `abstract def fields` to this just so it is extra clear what this module requires?